rma/migrations/16.0.1.4.0/post-migration.py

```diff
@@ -6,9 +6,12 @@
 
 @openupgrade.migrate()
 def migrate(env, version):
-    """Similar behavior to create_rma_routes of post_init_hook."""
+    """Similar behavior to create all the data of post_init_hook."""
```

Comment:
Maybe this comment was correct. Now re-reading it, it seems that it was a method name.